### PR TITLE
migrate session tests

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### CODE REVIEW

File: jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy [hunk 1]
```diff
@@ -165,11 +163,15 @@ def test_suites = { suites,option ->
...
-                        export log_level=error
-                        go install gotest.tools/gotestsum@latest
-                        gotestsum --format standard-verbose --junitfile "junit-report.xml" -- -with-tikv -pd-addrs=127.0.0.1:2379,127.0.0.1:2389,127.0.0.1:2399 -timeout 20m -vet=off ${option} '${suites}'
-                        #go test -with-tikv -pd-addrs=127.0.0.1:2379 -timeout 20m -vet=off
+                        if [ -d "${suites}" ]
+                        then
+                            cd ${suites}
+                            export log_level=error
+                            go install gotest.tools/gotestsum@latest
+                            gotestsum --format standard-verbose --junitfile "junit-report.xml" -- -with-tikv -pd-addrs=127.0.0.1:2379,127.0.0.1:2389,127.0.0.1:2399 -timeout 20m -vet=off
```
Comment (purelind) [inline jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy:171]: `-with-tikv` is correct? (for pessimistictest and realtikvtest)

test ci link: https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/77471/pipeline/78

Comment (tisonkun) [reply jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy]: You're right that tag name changed. Push a new commit to fix.

**Comment by purelind** [comment]
test ci link https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/77487/pipeline/76/

**Comment by purelind** [comment]
test ci link https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/77503/pipeline/78

**Pull request closed without merging**